.gitignore

```diff
@@ -68,3 +68,6 @@ yarn.lock
 !**/types/**/*.d.ts
 
 page-functions-test-case*out*.js
+package.json
+yarn.lock
+cli/test/smokehouse/frontends/smokehouse-bin.js
```

Comment:
Can you revert the changes to this file, smokehouse-bin.js, package.json and yarn.lock?